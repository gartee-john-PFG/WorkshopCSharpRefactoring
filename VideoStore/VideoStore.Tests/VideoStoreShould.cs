namespace VideoStore.Tests
{
    using System.Globalization;
    using System.Reflection.Context;
    using System.Reflection.Metadata;
    using ApprovalTests;
    using ApprovalTests.Combinations;
    using ApprovalTests.Reporters;
    using Xunit;
    
    [UseReporter(typeof(QuietReporter))]
    public class VideoStoreShould
    {
        [Fact]
        public void ApproveCustomerStatement()
        {
            Assert.True(1 == 1);
        }
    }
}