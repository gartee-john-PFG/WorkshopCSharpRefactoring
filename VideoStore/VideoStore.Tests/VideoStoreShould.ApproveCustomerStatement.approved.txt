[Star Wars,0,1] => Rental Record for customer name
	Star Wars	2
Amount owed is 2
You earned 1 frequent renter points
[Star Wars,0,2] => Rental Record for customer name
	Star Wars	2
Amount owed is 2
You earned 1 frequent renter points
[Star Wars,0,3] => Rental Record for customer name
	Star Wars	3.5
Amount owed is 3.5
You earned 1 frequent renter points
[Star Wars,0,4] => Rental Record for customer name
	Star Wars	5
Amount owed is 5
You earned 1 frequent renter points
[Star Wars,1,1] => Rental Record for customer name
	Star Wars	3
Amount owed is 3
You earned 1 frequent renter points
[Star Wars,1,2] => Rental Record for customer name
	Star Wars	6
Amount owed is 6
You earned 2 frequent renter points
[Star Wars,1,3] => Rental Record for customer name
	Star Wars	9
Amount owed is 9
You earned 2 frequent renter points
[Star Wars,1,4] => Rental Record for customer name
	Star Wars	12
Amount owed is 12
You earned 2 frequent renter points
[Star Wars,2,1] => Rental Record for customer name
	Star Wars	1.5
Amount owed is 1.5
You earned 1 frequent renter points
[Star Wars,2,2] => Rental Record for customer name
	Star Wars	1.5
Amount owed is 1.5
You earned 1 frequent renter points
[Star Wars,2,3] => Rental Record for customer name
	Star Wars	1.5
Amount owed is 1.5
You earned 1 frequent renter points
[Star Wars,2,4] => Rental Record for customer name
	Star Wars	3
Amount owed is 3
You earned 1 frequent renter points
